packages/app/universal-ts-utils/src/array/callChunked.ts

```diff
@@ -0,0 +1,11 @@
+
+export async function callChunked<Item>(
+  chunkSize: number,
+  array: readonly Item[],
+  processFn: (arrayChunk: Item[]) => Promise<unknown>,
+): Promise<void> {
```

Comment:
With this one, all array utils from node-core are already here, so we plan removing them from there